configure.ac

```diff
@@ -630,13 +622,8 @@ fi
 
 AC_CHECK_FUNCS([mblen], [], [HAVE_MBLEN=1], [])
 
-rm -f Makefile
-if test "$enable_maintainer_mode" = yes ; then
-	AC_MSG_NOTICE(creating maintainer Makefile)
-	ln -s maintainer.mak Makefile
-else
-	AC_CONFIG_FILES(Makefile ctags.1)
-fi
+AC_CONFIG_FILES([Makefile])
+AC_CONFIG_FILES([ctags.1])
```

Comment:
I don't mind much.  Actually I would've written
```Autoconf
AC_CONFIG_FILES([Makefile
                 ctags.1])
```
for readability (esp. for packages having quite a few output files) -- FTR `autoconf --trace` outputs the raw argument value so 2 lines, with indentation on the 2nd.

But really this is cosmetic and if you meant to do it for a reason I don't really care (although maybe the next person with low AC knowledge will wonder if there is a behavioral reason for this, maybe).